app/components/create-dex.jsx

```diff
@@ -0,0 +1,108 @@
+import { Component } from 'react';
+import { Link }      from 'react-router';
+import Modal         from 'react-modal';
+import { connect }   from 'react-redux';
+import { push }      from 'react-router-redux';
+
+import { AlertComponent } from './alert';
+import { ReactGA }        from '../utils/analytics';
+import { createDex }      from '../actions/dex';
+
+export class CreateDex extends Component {
```

Comment:
thoughts on calling this file `dex-create` and component `DexCreate`? it's more consistent with `dex` and `dex-preview` and `Dex` and `DexPreview` - plus then all the dex-related things are listed next to each other.